.github/workflows/ci.yml

```
@@ -1,28 +1,24 @@
----
-name: CI
+name: ci
+
 on: [push, pull_request]
+
 jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         node-version: [12, 14, 16, 17]
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2.4.0
-      - name: Use Node.js
-        uses: actions/setup-node@v2.5.1
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
-      - name: Restore cached dependencies
-        uses: actions/cache@v2.1.7
+      - uses: actions/cache@v2
         with:
           path: node_modules
           key: node-modules-${{ hashFiles('package.json') }}
-      - name: Install dependencies
-        run: npm install
-      - name: Run Tests
-        run: npm run ci
+      - run: npm install
+      - run: npm run ci
```

Could we rename this `script`?
I was confused at the first reading

```suggestion
      - run: npm run test:ci
```

---

not in scope for this PR